src/vuex/modules/auth/actions.js

```diff
@@ -10,7 +10,7 @@ export default {
     return new Promise((resolve, reject) => {
       kuzzle
         .unsetJwtToken()
-        .loginPromise('local', {username: data.username, password: data.password}, '4h')
+        .loginPromise('local', {username: data.username, password: data.password}, '2h')
```

Comment:
Why?

Reply:
I changed it for debugging and forget to put back original value, my bad